dataflux_pytorch/benchmark/checkpointing/multinode/train.py

```diff
@@ -180,7 +196,7 @@ def main(ckpt_dir_path: str, ckpt_restore_path: str = ""):
             devices=os.environ.get("NUM_DEVICES", 'auto'),
             num_nodes=num_nodes,
         )
-        trainer.fit(model, dataloader, ckpt_path=new_ckpt_dir_path)
+        trainer.fit(model, dataloader)
```

Comment:
Was this ckpt_path param unnecessary since it's provided to the strategy? This is validated for every strategy?

Reply:
Yes, passing the checkpoint path here causes the trainer to look for an existing checkpoint to restore from, which isn't the behavior we want since we manually restore a checkpoint in the next line.